debian/config.json

```diff
@@ -37,6 +37,7 @@
   },
   "Log": {
     "Level": "info",
-    "Format": "text"
+    "Format": "text",
+    "File": ""
```

Comment:
By default it's stderr, it could not work with systemd but works perfect with upstart